ts/blueprints/ember-cli-typescript/index.js

```diff
@@ -33,6 +53,7 @@ module.exports = {
     let isAddon = this.project.isEmberCLIAddon();
     let isMU = this._detectMU();
     let includes = isMU ? ['src'] : ['app', isAddon && 'addon'].filter(Boolean);
+    const isPods = this.pod;
```

Comment:
Since the default blueprint is normally run as part of `ember install`, I believe this flag will only ever be set if we have users explicitly re-run the generator afterwards with `ember g ember-cli-typescript --pod`. Is that something we want to encourage?

Reply:
I tested this manually with the `.embercli.js` file having the pods flag set to true. The scenario I'm thinking of is


```shell
ember new i-should-not-use-pods --pods # sets up .embercli.js with "pods": true
cd i-should-not-use-pods
ember install ember-cli-typescript

```

Reply:
Gotcha; I'd forgotten you could set a default there